packages/common/src/HttpClient.ts

```diff
@@ -44,4 +46,11 @@ export class HttpClient {
     this.logger.info(`auditTransaction response: ${JSON.stringify(auditResponse)}`)
     return auditResponse
   }
+
+  async getNetworkConfiguration (): Promise<ConfigResponse> {
+    const gsnUrl = `https://raw.githubusercontent.com/opengsn/gsn-networks/${gsnRuntimeVersion}/client-config.json`
```

Comment:
Why not request a flat configuration JSON for a specific network? Why get all networks' configurations? Most Dapps know very well where they are deployed and don't need to support any other network. I would say the only exception is CTF.